.gitignore

```diff
@@ -27,6 +27,8 @@ share/python-wheels/
 *.egg
 MANIFEST
 
+# Exported strings & generated translation templates
+src/frontend/i18n/frontend.json
```

Comment:
When I execute `make build-front`, I end up with non-versioned files : 
```
$ git status
On branch feat/moderator-cleaning
Your branch is up to date with 'origin/feat/moderator-cleaning'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   src/frontend/js/translations/es-ES.json
        modified:   src/frontend/js/translations/fr-CA.json
        modified:   src/frontend/js/translations/fr-FR.json
        modified:   src/frontend/js/translations/fr.json
```

Maybe something is missing in the `.gitignore` ?

Reply:
with the make lint-front it gets sorted

Reply:
Files are now ignored it git !